src/libs/antares/optimization-options/include/antares/optimization-options/options.h

```diff
@@ -28,16 +28,24 @@ struct SingleOptimOptions
 {
     std::string solverName = "sirius";
     std::string solverParameters;
+
+    // Reusing basis of first optimization (in case we have 2 weekly
+    // linear optimizations [not MILP])
+    bool solverAcquiresBasis = false;
```

Comment:
`solverUsesBasis` seems more explicit for me (with acquire I am a bit confused of wheter it is storage or usage)

Reply:
Done